package.json

```diff
@@ -1,70 +1,71 @@
 {
-    "main": "node_modules/expo/AppEntry.js",
```

Comment:
It would be great if you could run `npx prettier -w .` in the project directory and then commit the changes.

I think we should avoid commiting 30.000 changes each time we create a new PR because of the styling of `package.json` and `package-lock.json` files.

Reply:
thx for the feedback @DominicLuidold. I have removed the es lint warnings from our pr and also ran the prettier command :)